### PR TITLE
[Konvoy 2.x] Uninstall Note

### PR DESCRIPTION
**Pull request by mwrightd2**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-78963

## Description of changes being made

Added a note in 2.x Choose Infrastructure documentation mentioning that uninstall steps are listed per environment.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-<add_pr_##_here>.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by mmcelroy91 — APPROVED** [review]

File: pages/dkp/konvoy/2.0/choose_infrastructure/index.md [hunk 1]
```diff
@@ -10,6 +10,6 @@ enterprise: false
 
 When installing Konvoy for a project, line-of-business, or enterprise, the first step is to determine the infrastructure on which you want to deploy.
 
-The infrastructure you select then determines the specific requirements for a successful installation.
+The infrastructure you select then determines the specific requirements for a successful installation. If you have decided to uninstall Konvoy, refer to the same infrastructure documentation you have selected for specific steps to take down your cluster.
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.0/choose_infrastructure/index.md:13]:
```suggestion
The infrastructure you select then determines the specific requirements for a successful installation. If you decide to uninstall Konvoy, refer to the same infrastructure documentation you selected for the specific steps to take down your cluster.
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/index.md [hunk 2]
```diff
@@ -10,6 +10,6 @@ enterprise: false
 
 When installing Konvoy for a project, line-of-business, or enterprise, the first step is to determine the infrastructure on which you want to deploy.
 
-The infrastructure you select then determines the specific requirements for a successful installation.
+The infrastructure you select then determines the specific requirements for a successful installation. If you have decided to uninstall Konvoy, refer to the same infrastructure documentation you have selected for specific steps to take down your cluster.
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/choose-infrastructure/index.md:14]: ditto from above

**Review by Colin1841 — APPROVED** [review]

File: pages/dkp/konvoy/2.2/choose-infrastructure/index.md [hunk 3]
```diff
@@ -10,6 +10,6 @@ enterprise: false
 
 When installing DKP Konvoy for a project, line-of-business, or enterprise, the first step is to determine the infrastructure on which you want to deploy.
 
-The infrastructure you select then determines the specific requirements for a successful installation.
+The infrastructure you select then determines the specific requirements for a successful installation. If you have decided to uninstall DKP Konvoy, refer to the same infrastructure documentation you have selected for specific steps to take down your cluster.
 
-The different types of infrastructures supported in Konvoy are listed below:
+The different types of infrastructures supported in DKP Konvoy are listed below:
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.2/choose-infrastructure/index.md:15]: Do we want this to be `DKP Konvoy` - or should it just be `DKP`? - honestly don't know what's best: 

```suggestion
The different types of infrastructures supported in DKP are listed below:
```

**Review by msdolbey — APPROVED** [review]
One small nit, but don't know if it's neccessary even.

👍 
Improvements!

**Pull request merged**